e2e-tests/docker-compose.yaml

```diff
@@ -38,3 +47,11 @@ services:
     environment:
       - hapi.fhir.fhir_version=R4
       - hapi.fhir.partitioning.allow_references_across_partitions=false
+  enrollment:
+    # building the image now as enrollment is not yet merged, as well, it needs to test the latest changes
+    # image: ${ENROLLMENT_IMAGE:-ghcr.io/santeonnl/enrollment_form:main}
```

Comment:
can we fix this before merging?

And, when the e2e test does not use this, we don't need the container in the setup yet i.m.o.

Reply:
I could change the test-case to only validate the code sends out a redirect, but not actually perform the redirect. 

I figured to add this already as we probably want to validate the complete flow in the e2e runs. The enrollment will mainly function as the front-end application for the orchestrator. Shall I take the code out for now and put it back in when we've changed the Task-notification + sub-Task.input & sub-Task.output?